openeogeotrellis/backend.py

```diff
@@ -2289,17 +2288,10 @@ def as_arg_element(dependency: dict) -> dict:
 
             eodata_mount = "/eodata2" if use_goofys else "/eodata"
 
-            jinja_path = pkg_resources.resource_filename(
-                "openeogeotrellis.deploy", "sparkapplication.yaml.j2"
-            )
-            jinja_dir = os.path.dirname(jinja_path)
-            jinja_template = Environment(
-                loader=FileSystemLoader(jinja_dir)
-            ).from_string(open(jinja_path).read())
-
             spark_app_id = k8s_job_name()
 
-            rendered = jinja_template.render(
+            sparkapplication_dict_ = k8s_render_manifest_template(
```

Comment:
that trailing underscore in `sparkapplication_dict_` is not necessary anymore (it was originally with `dict_` to avoid overwriting the `dict` built-in)